plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -330,6 +330,11 @@ export class PersonState {
             return
         }
 
+        // The first distinct_id that is used in identify or alias (not the alias_id nor anon_id)
```

Comment:
This comment in this spot is not useful IMO - it adds more confusion than it resolves since it talks about things not relevant on the next lines.

I recommend removing it or e.g. documenting what is_identified column means elsewhere in the code.